scalopus_tracing/include/scalopus_tracing/trace_configurator.h

```diff
@@ -44,14 +44,22 @@ class TraceConfigurator
 {
 private:
   TraceConfigurator();
+  TraceConfigurator(const TraceConfigurator&) = delete;
```

Comment:
Why not `TraceConfigurator(const TraceConfigurator&&) = delete;` as well?

Reply:
I missed that, thanks :+1: